espnet2/gan_svs/uhifigan/uhifigan.py

```diff
@@ -132,7 +134,7 @@ def __init__(
                 getattr(torch.nn, nonlinear_activation)(**nonlinear_activation_params),
                 torch.nn.Dropout(dropout),
             )
-        hidden_channels = channels
```

Comment:
Is this OK?

Reply:
hidden_channels is no used at any part of the code, so this assignment is unused.

Reply:
Confirmed, this change should be fine! Thanks!